packages/job-manager/src/worker/worker.spec.ts

```diff
@@ -63,7 +63,7 @@ describe(Worker, () => {
                     activeToRunningTasksRatio: activeToRunningTasksRatioDefault,
                     addTasksIntervalInSeconds: addTasksIntervalInSecondsDefault,
                     maxWallClockTimeInHours: maxWallClockTimeInHours,
-                };
+                } as JobManagerConfig;
```

Comment:
same here
can you add a value for "sendNotificationTasksCount" instead of casting, and then check the value in the UTs something like: **expect(poolLoadGenerator.sendNotificationTasksCount).toEqual(x);**